06-pde-constrained-optimisation.ipynb

```diff
@@ -113,11 +129,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "try:\n",
-    "    mesh = Mesh(\"stokes-control.msh\")\n",
-    "except FileNotFoundError:\n",
-    "    !wget \"https://raw.githubusercontent.com/firedrakeproject/ICCS-Cam/main/stokes-control.msh\" -O \"stokes-control.msh\"\n",
-    "    mesh = Mesh(\"stokes-control.msh\")\n"
+    "mesh = Mesh(\"stokes-control.msh\")"
```

Comment:
I think this needs to have a `try: ... except: ...` block to work with colab, but the URL needs updating

Reply:
`https://raw.githubusercontent.com/firedrakeproject/notebooks/main/stokes-control.msh`